rr_platform/launch/sedani_lidar.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+    <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
+    <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
```

Comment:
You should probably set up udev rules for this. It will not always be USB0 if there are other USB devices. IDK how many of those you have.